quickstarts/microsoft.sql/sql-logical-server-aad-only-auth/azuredeploy.json

```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.1",
+    "parameters": {
+        "server": {
+            "type": "string",
+            "defaultValue": "[uniqueString('sql', resourceGroup().id)]",
+            "metadata": {
+                "description": "The name of the SQL logical server."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all resources."
+            }
+        },
+        "aad_admin_name": {
+            "type": "String",
+            "metadata": {
+                "description": "The name of the Azure AD admin for the SQL server."
+            }
+        },
+        "aad_admin_objectid": {
+            "type": "String",
+            "metadata": {
+                "description": "The Object ID of the Azure AD admin."
+            }
+        },
+        "aad_admin_tenantid": {
+            "type": "String",
+            "metadata": {
```

Comment:
add a defaultValue of "[subscription().tenantId]" and remove the parameter value from the parameters file

Reply:
Thanks! I'll make this change